config/application.rb

```diff
@@ -0,0 +1,26 @@
+require File.expand_path('../boot', __FILE__)
+
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module SideciRailsSandbox
+  class Application < Rails::Application
+    # Settings in config/environments/* take precedence over those specified here.
```

Comment:
__[staging][rubocop]__
* Metrics/LineLength: Line is too long. [82/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)